Doc/library/stdtypes.rst

```diff
@@ -1010,11 +1010,11 @@ operations have the same priority as the corresponding numeric operations. [3]_
 | ``x not in s``           | ``False`` if an item of *s* is | \(1)     |
 |                          | equal to *x*, else ``True``    |          |
 +--------------------------+--------------------------------+----------+
-| ``s + t``                | the concatenation of *s* and   | (6)(7)   |
+| ``s + t`` or ``s += t``  | the concatenation of *s* and   | (6)(7)   |
 |                          | *t*                            |          |
 +--------------------------+--------------------------------+----------+
 | ``s * n`` or             | equivalent to adding *s* to    | (2)(7)   |
-| ``n * s``                | itself *n* times               |          |
+| ``n * s`` or ``s *= n``  | itself *n* times               |          |
```

Comment:
I don't think the second form is *an equivalent* of `n * s`.  Ditto for concatenation.

Rather you could extend the footnote to mention that some sequence types have support for in-place concatenation (or repetition).

CC @StanFromIreland